doc/user/spark.md

```diff
@@ -19,21 +21,25 @@ Spark. You can ignore this section if you already have the pre-built jars.
 After build, the package daos-java-<version>-assemble.tgz will be available under distribution/target.
 
 ## Deploy DAOS Hadoop Filesystem Jars
+
 After unzipping daos-java-<version>-assemble.tgz, you will get the following files.
-* daos-java-api-<version>.jar and hadoop-daos-<version>.jar
+* daos-client-<version>.jar and hadoop-daos-<version>.jar
+
```

Comment:
:nit `daos-java-<version>-assemble.tgz` will be put in github release -> assets?
Why not just put 2 separate jars?

Reply:
It's better for release since there are not only jars but daos-site-example.xml.